xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -226,13 +243,13 @@ namespace XBMCAddon
       /// @brief \python_func{ Decrypt(cipherKeyId, input, iv) }
       ///-----------------------------------------------------------------------
       /// \anchor xbmcdrm_Decrypt
-      /// Sets a system specific property value in the DRM system
+      /// Decrypt an encrypted data by using session keys.
       ///
-      /// @param      [byte] cipherKeyId Encryption key id (provided from a service handshake)
-      /// @param      [byte] input       Cipher text to decrypt
-      /// @param      [byte] iv          Initialization vector of cipher text
+      /// @param     cipherKeyId byte - Encryption key id (provided from a service handshake)
+      /// @param     input       byte - Cipher text to decrypt
+      /// @param     iv          byte - Initialization vector of cipher text
       ///
-      /// @return     Decrypted input data
+      /// @return    Decrypted input data
       ///
       ///------------------------------------------------------------------------
       /// @python_v18 New function added.
```

Comment:
`@python_v19 With python 3 all arguments need to be of type bytearray instead of byte.`

Reply:
done